pango/src/matrix.rs

```diff
@@ -0,0 +1,34 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{Matrix, Rectangle};
+use glib::translate::*;
+
+impl Matrix {
+    pub fn new(xx: f64, xy: f64, yx: f64, yy: f64, x0: f64, y0: f64) -> Self {
+        Self(ffi::PangoMatrix {
+            xx,
+            xy,
+            yx,
+            yy,
+            x0,
+            y0,
+        })
+    }
+
+    #[doc(alias = "pango_matrix_transform_pixel_rectangle")]
+    pub fn transform_pixel_rectangle(&self, mut rect: Option<&mut Rectangle>) {
```

Comment:
Why the rect needs to be mut?

Reply:
Because it's modified by the function

Reply:
Also why `Option`? While technically the function allows that, it has absolutely no effect if no rectangle is passed in :) That's both input and output of the function. Same for the other one.

Reply:
I thought that the function was doing something even with a NULL rectangle. Didn't check the source code though...